app/src/routes/routesMap.js

```diff
@@ -135,7 +127,7 @@ const routesMap = {
     },
   },
   [PROJECT_DETAILS_PAGE]: {
-    path: `/administrate/projects/:organizationSlug?/:projectKey/:projectSection(${MEMBERS}|${MONITORING})?`,
+    path: `/administrate/projects/organizations/:organizationSlug?/projects/:projectSlug/:projectSection(${MEMBERS}|${MONITORING})?`,
```

Comment:
Please also add a `TODO` note here as the administrate part will be changed to instance level.
All pages under it will be changed accordingly, f.e.
`/administrate/projects/:projectName/:projectSection(${MEMBERS}|${MONITORING})?`,
--->
`/organizations/:organizationSlug/projects/:projectSlug/:projectSection(${MEMBERS}|${MONITORING})?`

`/administrate/users`
--->
`/users`

Reply:
Done.